src/js/components/Header/PlayingBar.react.js

```diff
@@ -158,6 +159,14 @@ export default class PlayingBar extends Component {
 
     tick() {
         this.setState({ elapsed: Player.getAudio().currentTime });
+
+        if (!Player.isThresholdReached() && Player.getAudio().currentTime >= Player.getAudio().duration * Player.getThreshold()) {
```

Comment:
Maybe you can move `      Player.getAudio().currentTime >= Player.getAudio().duration * Player.getThreshold()` into local variable?